globalSetupFile.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Jest creates a copy of the global 'process' object and uses it instead of the one provided by node.
```

Comment:
instead of this, can we just have an integration test? We don't want `domain` in jest's own tests